hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/data/Data.java

```diff
@@ -92,7 +92,7 @@
 
     /** For JSON Construction ONLY */
     public Data() {
-        this(null, 0L, null);
+        this(null, System.currentTimeMillis(), null);
```

Comment:
Because in the REST endpoint, it is possible to create a Data object without timestamp without any fail, then when no timestamp field is found on the REST it uses this constructor setting the timestamp by default to 0 instead to current time.